tests/Unit/App/Services/LanguageLineServiceTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Tests\Unit\Services;
+
+use App\Models\LanguageLine;
+use App\Services\LanguageLineService;
+use Tests\TestCase;
+use ValueError;
+
+/**
+ * @internal
+ *
```

Comment:
I suggest you don't repeat yourself in the tests below with the annotations that are common:
```
 * 
 * @group App\Services
 * 
```

Reply:
Fixed